src/Library/demos/Scale/main.js

```diff
@@ -0,0 +1,33 @@
+const scale_one = workbench.builder.get_object("one");
+const scale_two = workbench.builder.get_object("two");
+const scale_disabled = workbench.builder.get_object("disabled");
+
+scale_two.add_mark(25.0, "left", "A");
+scale_two.add_mark(50.0, "right", "B");
+scale_two.add_mark(75.0, "right", "C");
+scale_two.add_mark(100.0, "right", "D");
+scale_two.set_increments(25.0, 25.0);
+
+scale_disabled.set_show_fill_level(25);
```

Comment:
`set_show_fill_level` takes a boolean, not an integer
makes sure that you're using the APIs correctly—not just that things work

also `show-fille-level` is a property so it can be moved to `main.blp` if needed

Reply:
okay, I'll move it to main.blp